src/cryptoadvance/specter/util/version.py

```diff
@@ -59,6 +59,37 @@ def loop(self, dt=3600):
             )
             time.sleep(dt)
 
+    def _check_if_version_is_available(self, custom_version):
```

Comment:
This check is checking on pypi. But the relevant check is the tag/branch in the specter-ext repo:
https://github.com/cryptoadvance/specterext-dummy/tags

This is what needs to be used when someone is referring to a tag. So can you change that to check whether the tag/branch is existing? You can also put that check as a class-method in the `GithubUrlLoader`, It doesn't need to be in the util package.

Reply:
Okay will do that